CtaReactiveMaster/Sources/Views/Home/HomeViewController.swift

```diff
@@ -38,52 +35,49 @@ final class HomeViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        fetchNewsAPI()
+        bindToViewModel()
+        viewModel.input.viewDidLoad()
+    }
 
+    private func bindToViewModel() {
         refreshControl
             .rx
             .controlEvent(.valueChanged)
             .asDriver()
             .drive { [weak self] _ in
-                self?.fetchNewsAPI()
+                self?.viewModel.input.pullToRefresh()
             }
```

Comment:
[Nits]
[63行目](https://github.com/CA21engineer/cta_reactive_master_answer/pull/20/files#diff-7f2d95d4b60c3aecb46f47f42caa1ffcdcdf5ee058beff3b286b81d54596a214R63
)のように改行ルール揃えるなら以下のように。
```suggestion
            }.disposed(by: disposeBag)
```